src/EditorFeatures/Core/Implementation/MetadataAsSource/MetadataAsSourceFileService.cs

```diff
@@ -253,12 +253,17 @@ private void RemoveDocumentFromWorkspace_NoLock(MetadataAsSourceGeneratedFileInf
             }
         }
 
-        private void InitializeWorkspace(Project project)
+        private void InitializeWorkspace(Project project, OptionSet option)
```

Comment:
@basoundr We have a "TestOptionServiceFactory" that you should be able to add to the composition used by the MAS tests.